### DISCUSSION TITLE
能不能调用微软azure的api做通知机器人

### DISCUSSION THREAD

**Discussion by chise0713** [jeessy2/ddns-go#707] [Q&A]
我在网上找了一圈，没找到该怎么做，我有个365账号想试试看

**Comment by WaterLemons2k** [comment]
Azure 没看见。有一个 Microsoft Teams 的：[创建传入 Webhook - Teams | Microsoft Learn](https://learn.microsoft.com/zh-cn/microsoftteams/platform/webhooks-and-connectors/how-to/add-incoming-webhook)
